### DISCUSSION TITLE
Shout out to cdubz

### DISCUSSION THREAD

**Discussion by lenaxia** [babybuddy/babybuddy#739] [General]
Just want to shout out for the great app. While I've stopped using it at 6 months, a friend of mine just had their 2nd and were planning on doing the ol' pen and paper like did for #1, but I convinced them to give babybuddy a go. 

I spun up an instance and set up the grafana dashboard I made and the ios shortcuts, for them and now their entire family is using it, mom, dad, grandma and mom's sister. 

Its helped them take the guess work out of what their newborn needs when he starts crying. 

So thanks again for the great app that makes a difficult time just a little bit easier.

**Comment by cdubz** [comment]
Thank you!! I have been woefully negligent lately with work and life stuff but will hopefully get back to more fun work on Baby Buddy in the not-too-distant future. Love to hear from users! ❤️
